vignettes/pkgdown/examples/salience.Rmd

```diff
@@ -14,20 +14,29 @@ The `ai_salience()` function allows you to classify documents based on their rel
 ```{r getting-started}
 library(quanteda)
 library(quanteda.llm)
-library(quanteda.tidy)
 
-data_corpus_inaugural <- tail(data_corpus_inaugural, 3)
+data_corpus_inaugural <- data_corpus_inaugural[57:60]
 ```
 
 ### Using `ai_salience()` for salience rating of topics
 
 ```{r ai_salience}
 # define the topics for salience classification
 topics <- c("economy", "environment", "healthcare")
-data_corpus_inaugural <- data_corpus_inaugural %>%
-  mutate(ai_salience(text, topics, chat_fn = chat_openai, model = "gpt-4o",
```

Comment:
@SeraphineM The reason I think this does not work is that the piping of `data_corpus_inaugural` feeds into the first argument of `mutate()`, not of `ai_salience()`. There could be a way to make the tidy piping work but it would require a new mutate.